code/modules/mob/mob.dm

```diff
@@ -1920,6 +1912,36 @@ mob/proc/on_foot()
 	user.transmogrify()
 	user.remove_spell(src)
 
+/obj/transmog_body_container
+	name = "transmog body container"
+	desc = "You should not be seeing this."
+	flags = TIMELESS
+	var/mob/contained_mob
+
+/obj/transmog_body_container/proc/set_contained_mob(var/mob/M)
+	if(M)
+		M.forceMove(src)
+		contained_mob = M
+
+/obj/transmog_body_container/proc/get_rid_of()
+	for(var/atom/movable/AM in contents)
+		AM.forceMove(get_turf(src))
+	contained_mob = null
+	qdel(src)
+
+/obj/transmog_body_container/proc/get_contained_mob()
```

Comment:
Why is this returning the first object in contents instead of just `contained_mob`?